apps/web/src/pages/tx/[hash].tsx

```diff
@@ -281,30 +281,28 @@ const Tx: NextPage = () => {
                 },
                 {
                   name: "Parent L2 block hash",
-                  value: (
-                    <div className="flex items-center gap-2">
-                      {"0x" + decodedData.parentL2BlockHash + "..."}
-                    </div>
-                  ),
+                  value: "0x" + decodedData.parentL2BlockHash + "...",
```

Comment:
I've found `Ox` several times repeated as a raw string. I would create a const for it

Reply:
Beginning with 0x just means that the string is hexadecimal. I feel like having const for just a two characters string is kind of an overkill